src/addcall.c

```diff
@@ -290,70 +287,39 @@ int addcall2(void) {
 	}
     }
 
-    if (!continentlist_only
-	    && exclude_multilist_type == EXCLUDE_CONTINENT) {
-	if (is_in_continentlist(dxcc_by_index(cty)->continent)) {
-	    excl_add_veto = 1;
-	}
-    }
-
-    if (exclude_multilist_type == EXCLUDE_COUNTRY) {
-	if (is_in_countrylist(cty)) {
-	    excl_add_veto = 1;
-	}
+    excl_add_veto |= check_veto(cty);
+    if (excl_add_veto) {
+	add_ok = 0;
+	new_cty = 0;
+	addcallarea = 0;
     }
 
     if (add_ok == 1) {
 
-	bandinx = log_get_band(lan_logline);
 	qsos_per_band[bandinx]++;
 
 	worked[station].band |= inxes[bandinx];	/* worked on this band */
 
 	if (excl_add_veto == 0) {
 
-	    switch (bandinx) {
-
-		case BANDINDEX_160:
-		case BANDINDEX_80:
-		case BANDINDEX_40:
-		case BANDINDEX_20:
-		case BANDINDEX_15:
-		case BANDINDEX_10:
-
-		    if (pfxnumcntidx < 0) {
-			if (cty != 0 && (countries[cty] & inxes[bandinx]) == 0) {
-			    countries[cty] |= inxes[bandinx];
-			    countryscore[bandinx]++;
-//                              new_cty = cty;
-			}
-			if (zone != 0 && (zones[zone] & inxes[bandinx]) == 0) {
-			    zones[zone] |= inxes[bandinx];
-			    zonescore[bandinx]++;
-//                              new_zone = zone;
-			}
-		    } else {
-			if ((pfxnummulti[pfxnumcntidx].qsos[pxnr] & BAND10) == 0) {
-			    pfxnummulti[pfxnumcntidx].qsos[pxnr] |= inxes[bandinx];
-			    addcallarea = 1;
-			    zonescore[bandinx]++;
-			    countryscore[bandinx]++;
-			}
-		    }
-		    break;
-
-		case BANDINDEX_30:
-		case BANDINDEX_17:
-		case BANDINDEX_12:
-
-		    if (cty != 0 && (countries[cty] & inxes[bandinx]) == 0) {
-			countries[cty] |= inxes[bandinx];
-		    }
-		    if (zone != 0 && (zones[zone] & inxes[bandinx]) == 0) {
-			zones[zone] |= inxes[bandinx];
-		    }
-		    break;
-
+	    if (pfxnumcntidx < 0) {
+		if (cty != 0 && (countries[cty] & inxes[bandinx]) == 0) {
+		    countries[cty] |= inxes[bandinx];
+		    countryscore[bandinx]++;
+//                  new_cty = cty;
+	    }
```

Comment:
strange indent, maybe astyle is need

Reply:
Good catch. Vi's autoindent had difficulties with the commented out line above.